components/metadata.tsx

```diff
@@ -8,24 +8,27 @@ import mZAR from "../images/mZAR-coin.png";
 
 export const Metadata = () => {
   return (
-    <div id="asset-metadata" className={"mb-20  grid px-4 lg:grid-cols-4"}>
+    <div
+      id="asset-metadata"
+      className={"mb-40  grid px-4 md:px-20 lg:grid-cols-3 2xl:grid-cols-3 lg:-mt-32 xl:-mt-52 2xl:-mt-96"}
+    >
       <div className={""}></div>
       <div className={"lg:w-[600px]"}>
-        <div className={"lg:py-8"}>
+        <div className={"md:py-10 lg:py-8"}>
           <div
             className={
-              " pb-8 text-center font-poppins text-3xl  text-[#DE1B55]"
+              " pb-8 lg:pb-0 text-center  font-poppins text-3xl  text-[#DE1B55]"
             }
           >
-            <h2>
-              Asset<strong className="pl-2">Metadata</strong>
+            <h2 className="font-[300]">
```

Comment:
is there no tailwind size that matches 300?

Reply:
Updated where applicable